INFO     backend.layers.business.business:business.py:511 {'message': 'ingesting dataset', 'collection_version_id': e67be74b-e094-44f5-af4e-ca8903636816, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:511 {'message': 'ingesting dataset', 'collection_version_id': e67be74b-e094-44f5-af4e-ca8903636816, 'url': None, 'current_dataset_version_id': 650a38c7-af36-44e6-b837-81c92dcefb95}
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [6cccfec2-32a4-4b2b-b732-2684f567bdae/raw.h5ad] to s3://artifact_bucket/6cccfec2-32a4-4b2b-b732-2684f567bdae/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:244 Main: Starting thread for h5ad update
INFO     processing:dataset_metadata_update.py:264 Main: Starting thread for rds update
ERROR    processing:dataset_metadata_update.py:302 Cannot find cxg artifact uri for 650a38c7-af36-44e6-b837-81c92dcefb95.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status